tekton/publish.yaml

```diff
@@ -140,8 +140,7 @@ spec:
       done
 
       # Rewrite "devel" to inputs.params.versionTag
-      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/pipeline/config/controller.yaml
-      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/pipeline/config/webhook.yaml
+      sed -i 's/devel/$(inputs.params.versionTag)/g' /workspace/go/src/github.com/tektoncd/pipeline/config/*.yaml
```

Comment:
Would this be to paranoid?

```
sed -i -e 's/\(pipeline.tekton.dev\/release\): "devel"/\1: "$(inputs.params.versionTag)"/g' -e 's/\(version\): "devel"/\1: "$(inputs.params.versionTag)"/g' /workspace/go/src/github.com/tektoncd/pipeline/config/*.yaml
```

Reply:
maybe :upside_down_face: but I am fine having it :wink:

Reply:
done